backend_api/README.md

```diff
@@ -89,6 +89,7 @@ Start your virtual environment:
 Install the dependencies:
 
     pip install -r requirements.txt
+    (Other dependencies are flas_restx, google-cloud)
```

Comment:
Please add correct instructions to install these dependencies. If they can be added to the requirements.txt please do so.